modules/social_features/social_event/social_event.install

```diff
@@ -1800,66 +1800,83 @@ function social_event_update_8915() {
 }
 
 /**
- * Add the 'enable event enrolment' field.
- */
-function social_event_update_8916() {
```

Comment:
We can't just remove update hooks that were released before.
You can remove the body of the update hook if it is deprecated, but leave a clear comment on what was done. 
See an example in `social_core_update_10102()`